docs/Makefile

```
@@ -35,11 +35,17 @@ html:
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
 doctest:
-	cd ../rust && $(CARGO) test --doc
-	$(SPHINXBUILD) $(SPHINXOPTS) -b doctest source $(BUILDDIR)/doctest
+	make doctest-rust
+	make doctest-python
 	@echo
 	@echo "Doctest finished. The output is in $(BUILDDIR)/doctest/output.txt."
 
+doctest-rust:
+	cd ../rust && $(CARGO) test --doc
+
+doctest-python:
```

Dependencies are your friend!
```suggestion
doctest-python: html
```